### PR TITLE
Changes in styles

### PR DESCRIPTION
**Pull request by tonygoldcrest**
1. Changed color to currentColor.
2. Made border for the host element.
3. Changed height from pixels to em.

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-progress-bar/22)

### CODE REVIEW

File: vaadin-progress-bar.html [hunk 1]
```diff
@@ -13,15 +13,12 @@
   <template>
     <style>
       :host {
-        height: 3px;
-      }
-
-      [part="bar"] {
-        background: #ddd;
+        height: 0.5em;
```
Comment (jouni) [inline vaadin-progress-bar.html]: I would rather have this on the host element, so it’s easy to override. Do the inner parts adapt to the height of the host, btw?

**Comment by limonte** [comment]
Screenshots for visual tests should be updated:

```sh
gemini update test/visual
```

PS. I didn't know about `currentColor`, now the default theme is perfectly accessible for people with low vision :+1:

**Comment by tonygoldcrest** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[vaadin-progress-bar.html, line 16 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-progress-bar/22#-KwZeLSlO4GaPHYHdb01-r1-16:-KwdR7IASQssT3VM5DJg:b-jxge8c) ([raw file](https://github.com/vaadin/vaadin-progress-bar/blob/0efb95cd860ee4c08e880a81e504b577f277b3fb/vaadin-progress-bar.html#L16)):*
<details><summary><i>Previously, jouni (Jouni Koivuviita) wrote…</i></summary><blockquote>

I would rather have this on the host element, so it’s easy to override. Do the inner parts adapt to the height of the host, btw?
</blockquote></details>

Isn't it already on the host element?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-progress-bar/22)*

**Comment by tonygoldcrest** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[vaadin-progress-bar.html, line 16 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-progress-bar/22#-KwZeLSlO4GaPHYHdb01-r1-16:-KwdRMI_1MZT-TvuXJUZ:b-y5ddv4) ([raw file](https://github.com/vaadin/vaadin-progress-bar/blob/0efb95cd860ee4c08e880a81e504b577f277b3fb/vaadin-progress-bar.html#L16)):*
> Do the inner parts adapt to the height of the host, btw?
part="bar" and part="value" have height 100%

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-progress-bar/22)*

**Comment by limonte** [comment]
Sorry, I forgot to mention that Chrome is in separated config https://github.com/vaadin/vaadin-progress-bar/blob/master/.gemini-chrome.yml

In order to update Chrome screenshot you need to run:

```sh
gemini update test/visual -c .gemini-chrome.yml
```

**Comment by tonygoldcrest** [comment]
@limonte, done.

---

Review status: 0 of 7 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-progress-bar/22#-:-KwdWpGheHs3242aPt95:bhkal4j)*

File: vaadin-progress-bar.html (hunk 1 above)
Comment (jouni) [reply vaadin-progress-bar.html]: Aw, fff. I’m stupid, I read the diff completely wrong. Sorry. Nevermind 😆

**Comment by jouni** [comment]
Reviewed 1 of 1 files at r1, 1 of 1 files at r2, 5 of 5 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-progress-bar/22)*

**Comment by limonte** [comment]
:lgtm_strong: 

---

Reviewed 1 of 1 files at r1, 1 of 1 files at r2, 5 of 5 files at r3.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-progress-bar/22#-:-KwdzrS9D-sYcZD8uGoZ:b-uwkowd)*

**Pull request merged**